src/librbd/cache/WriteLogImageDispatch.cc

```diff
@@ -64,14 +54,11 @@ bool WriteLogImageDispatch<I>::read(
     return true;
   }
 
-  start_in_flight_io(aio_comp);
+  m_plugin_api.start_in_flight_io(aio_comp);
 
-  aio_comp->set_request_count(1);
-  aio_comp->read_result = std::move(read_result);
-  aio_comp->read_result.set_image_extents(image_extents);
+  m_plugin_api.update_aio_comp(aio_comp, 1, read_result, image_extents);
```

Comment:
Nit: you can probably combine the start w/ this method

Reply:
DONE